### PR TITLE
SEO updated

### PR DESCRIPTION
**Pull request by mmahalwy**

### CODE REVIEW

File: server/config/sitemap.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import sitemap from 'sitemap';
+
+const urls = [];
+
+Array.apply(null, {length: 114}).forEach((_, index) => {
+  const surahId = index;
+
+  urls.push({
```
Comment (sharabash) [inline server/config/sitemap.js]: consider also adding a url for each ayah

File: src/scripts/components/Html.js [hunk 2]
```diff
@@ -36,6 +36,29 @@ class Html extends React.Component {
               }}
               charSet="UTF-8"
             />
+            {/* SEO: https://developers.google.com/structured-data/slsb-overview#markup_examples */}
```
Comment (sharabash) [inline src/scripts/components/Html.js]: nice, this should give us a search field in google (if google allows it)

File: server/config/sitemap.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    { url: '/contactus',  changefreq: 'monthly', priority: 0.3 },
+    { url: '/contact',  changefreq: 'monthly', priority: 0.3 },
+    { url: '/donations',  changefreq: 'monthly', priority: 0.3 },
+    { url: '/contributions',  changefreq: 'monthly', priority: 0.3 },
+
+    { url: '/search',  changefreq: 'weekly',  priority: 0.7 }
+  ]
+});
+
+export default (server) => {
```
Comment (sharabash) [inline server/config/sitemap.js]: this process makes my cpu spin up like crazy, is it going to do this every time i run `npm run dev` or every time the server restarts? can you disable this if we're in development

**Comment by sharabash** [comment]
check comments, then its okay to merge :+1:

File: server/config/sitemap.js (hunk 3 above)
Comment (sharabash) [inline server/config/sitemap.js]: at least i think thats what it was

Comment (sharabash) [inline server/config/sitemap.js]: shrug i dont know why it would in retrospect

File: src/scripts/components/Html.js (hunk 2 above)
Comment (mmahalwy) [inline src/scripts/components/Html.js]: yep!

File: server/config/sitemap.js (hunk 1 above)
Comment (mmahalwy) [inline server/config/sitemap.js]: +1 to that :) I can most definitely do that.

File: server/config/sitemap.js (hunk 3 above)
Comment (mmahalwy) [inline server/config/sitemap.js]: I don't think this is it. It might go crazy when you hit this route and we have all the ayahs!

**Comment by mmahalwy** [comment]
@sharabash awesome! Going to make the update for each ayah

**Pull request merged**